src/index.js

```diff
@@ -66,13 +66,11 @@ const Dot = styled.button([], {
   width: '8px',
   height: '8px',
   color: 'inherit',
-  '&:focus': {
-    outline: 'none',
-    boxShadow: '0 0 0 1px'
-  }
-},
+  outline: 'none'
+  },
   props => ({
-    opacity: props.active ? 0.5 : 0.125
+    opacity: props.active ? 0.5 : 0.125,
+    boxShadow: props.focus ? '0 0 0 1px' : 'none'
```

Comment:
Would this be confusing to change the browser's focus behavior? Not sure that it would be expected to navigate to a link with the tab key if that's what this is meant to do?

Reply:
I think I did not understand you. This only determine the correct value for boxShadow for the button (Dot) with the index of the current slide displayed. The browser's focus behavior was not working well moving slides with the arrow keys.

Reply:
The tab key and shift+tab should cycle through focusable elements, then return key should select the slide. I believe it's working as expected on my end?